ledger/catchpointtracker.go

```diff
@@ -764,8 +790,14 @@ func (ct *catchpointTracker) createCatchpoint(ctx context.Context, accountsRound
 	var labelMaker ledgercore.CatchpointLabelMaker
 	var version uint64
 	params := config.Consensus[blockProto]
-	if params.EnableCatchpointsWithSPContexts {
-		labelMaker = ledgercore.MakeCatchpointLabelMakerCurrent(round, &blockHash, &dataInfo.TrieBalancesHash, dataInfo.Totals, &dataInfo.StateProofVerificationHash)
+	if params.EnableCatchpointsWithOnlineAccounts {
```

Comment:
Part of me thinks these different consensus params should be superseded by the catchpoint version number to simplify the checks like in this block.